install/lng/english.lng.php

```diff
@@ -63,6 +63,15 @@
 $lng['install']['admin_pass2'] = 'Administrator-Password (confirm)';
 $lng['install']['activate_newsfeed'] = 'Enable the official newsfeed<br><small>(https://inside.froxlor.org/news/)</small>';
 $lng['install']['serversettings'] = 'Server settings';
+$lng['install']['distribution'] = 'Distribution';
+$lng['install']['jessie'] = 'Debian Jessie';
```

Comment:
too hardcoded, you can get the distro names from the xmls

Reply:
ok